packages/core/src/service/plugin/classes.ts

```diff
@@ -31,12 +31,14 @@ export type Plugins = {
   layout?: LayoutPluginConfig[];
   content?: ContentPluginConfig[];
   native?: NativeFactory;
+  missingPlugin?: PluginComponentType;
```

Comment:
can you rename it to: `MissingPluginComponent` ? to highlight that you need to define a `Component` here.